src/ScheduleService.Mappers/Db/Interfaces/IDbTaskMapper.cs

```diff
@@ -1,11 +1,12 @@
 ﻿using LT.DigitalOffice.Kernel.Attributes;
 using LT.DigitalOffice.ScheduleService.Models.Db;
 using LT.DigitalOffice.ScheduleService.Models.Dto.Requests.Tasks;
+using System;
 
 namespace LT.DigitalOffice.ScheduleService.Mappers.Db.Interfaces;
 
 [AutoInject]
 public interface IDbTaskMapper
 {
-  DbTask Map(CreateTaskRequest request);
+  DbTask Map(CreateTaskRequest request, Guid id);
```

Comment:
It is better to write which id: like userId or creatorId

Reply:
fixed it